internal/attestation/measurements/measurements.go

```diff
@@ -330,6 +330,15 @@ func (m *M) UnmarshalYAML(unmarshal func(any) error) error {
 	return nil
 }
 
+// String returns a string representation of the measurements.
+func (m M) String() string {
+	returnString := ""
+	for i, measurement := range m {
+		returnString += fmt.Sprintf("%d: 0x%s\n", i, hex.EncodeToString(measurement.Expected))
+	}
+	return returnString
```

Comment:
```suggestion
	var returnString string
	for i, measurement := range m {
		returnString = strings.Join([]string{returnString, fmt.Sprintf("%d: 0x%s", i, hex.EncodeToString(measurement.Expected)}, ",")
	}
	return returnString
```
As mentioned before, I would prefer to keep the logging to one message per line.